Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/EwsExtensionEXOPowershellV3.ps1

```diff
@@ -1417,6 +1491,290 @@ class ExchangeOnlinePowershellV3Client
         #>
     }
 
+    [PSObject]DisableRule(
+    [string]$mailbox,
+    [string]$identity
+    )
+    {
+        $response = ""
+        try {
+            # Establish session to remote
+            $this.CreateSession()
+            # Import and Execute command
+            $cmd_params = @{ }
+            if ($mailbox) {
+                $cmd_params.Mailbox = $mailbox
+            }
+
+            if ($identity) {
+                $cmd_params.Identity = $identity
+            }
+            $response = Disable-InboxRule -Confirm:$false @cmd_params -WarningAction:SilentlyContinue
+        } finally {
+            $this.DisconnectSession()
+        }
+        return $response
+        <#
+        .DESCRIPTION
+        Disable an existing inbox rule in a given mailbox.
+
+        .PARAMETER mailbox
+        The mailbox that contains the Inbox rule.
+
+        .PARAMETER identity
+        Specifies the Inbox rule that you want to disable.
+
+        .EXAMPLE
+        Disable-InboxRule -Identity "MoveAnnouncements" -Mailbox "Joe@Contoso.com"
+
+        .OUTPUTS
+        PSObject - Raw response
+
+        .LINK
+        https://learn.microsoft.com/en-us/powershell/module/exchange/disable-inboxrule?view=exchange-ps
+        #>
+    }
+
+    [PSObject]EnableRule(
+    [string]$mailbox,
+    [string]$identity
+    )
+    {
+        $response = ""
+        try {
+            # Establish session to remote
+            $this.CreateSession()
+            # Import and Execute command
+            $cmd_params = @{ }
+            if ($mailbox) {
+                $cmd_params.Mailbox = $mailbox
+            }
+
+            if ($identity) {
+                $cmd_params.Identity = $identity
+            }
+            $response = Enable-InboxRule @cmd_params -WarningAction:SilentlyContinue
+        } finally {
+            $this.DisconnectSession()
+        }
+        return $response
+        <#
+        .DESCRIPTION
+        Enable an existing inbox rule in a given mailbox.
+
+        .PARAMETER mailbox
+        The mailbox that contains the Inbox rule.
+
+        .PARAMETER identity
+        Specifies the Inbox rule that you want to enable.
+
+        .EXAMPLE
+        Enable-InboxRule "Move To Junk Mail" -Mailbox "User 1"
+
+        .OUTPUTS
+        PSObject - Raw response
+
+        .LINK
+        https://learn.microsoft.com/en-us/powershell/module/exchange/enable-inboxrule?view=exchange-ps
+        #>
+    }
+
+    [PSObject]ListMailFlowRules()
+    {
+        $response = ""
+        try {
+            # Establish session to remote
+            $this.CreateSession()
+            # Import and Execute command
+            $response = Get-TransportRule -WarningAction:SilentlyContinue
```

Comment:
We might want to add a limit in list commands - so we won't bring all results if there are many. Is the API support limit/pagination?

Reply:
added